aea/test_tools/test_cases.py

```diff
@@ -182,6 +184,29 @@ def _run_python_subprocess(self, *args: str) -> subprocess.Popen:
         self.subprocesses.append(process)
         return process
 
+    def is_present_in_output(
+        self,
+        process: subprocess.Popen,
+        string: str,
+        timeout: int = DEFAULT_PROCESS_TIMEOUT,
+    ) -> bool:
+        """
+        Check if string present in agent process output.
+
+        :param process: a subprocess object to communicate.
+        :param string: a substring expected to appear in process output.
```

Comment:
we want to check for a comma separated list of strings not just a single string.

Reply:
Will upgrade, makes sence.